tests/zquantum/core/circuits/_unitary_test.py

```diff
@@ -1,51 +1,59 @@
-import cirq
 import numpy as np
 import pytest
 import sympy
-from zquantum.core.circuits import (
-    RX,
-    RY,
-    RZ,
-    XX,
-    XY,
-    YY,
-    Circuit,
-    H,
-    I,
-    X,
-    Y,
-    Z,
-    export_to_cirq,
-)
+from zquantum.core.circuits import RX, RY, RZ, XX, XY, YY, Circuit, H, I, X, Y, Z
 
 
 class TestCreatingUnitaryFromCircuit:
+    @pytest.fixture
+    def cirq_unitaries(self):
+        """
+        Note: We decided to go with file-based approach after extracting cirq
+        from z-quantum-core and not being able to use `export_to_cirq` anymore.
+        """
+        path_to_array = (
+            "/".join(__file__.split("/")[:-1]) + "/hardcoded_cirq_unitaries.npy"
+        )
+        return np.load(path_to_array, allow_pickle=True)
+
     @pytest.mark.parametrize(
-        "circuit",
+        "circuit, unitary_index",
         [
             # Identity gates in some test cases below are used so that comparable
             # Cirq circuits have the same number of qubits as Zquantum ones.
-            Circuit([RX(np.pi / 5)(0)]),
-            Circuit([RY(np.pi / 2)(0), RX(np.pi / 5)(0)]),
-            Circuit([I(1), I(2), I(3), I(4), RX(np.pi / 5)(0), XX(0.1)(5, 0)]),
-            Circuit(
-                [
-                    XY(np.pi).controlled(1)(3, 1, 4),
-                    RZ(0.1 * np.pi).controlled(2)(0, 2, 1),
-                ]
+            (Circuit([RX(np.pi / 5)(0)]), 2),
```

Comment:
Can you sort these so that `unitary_index` as same as index of items in this list :) ?